### ISSUE TITLE
taxclass values don't show on products filters

### ISSUE THREAD

**Issue by Netmees** [tsolucio/corebos#1495]
I wanna filter my products by tax and i realice that cant be done 

when you set a tax for a product it does not add taxclass value so it can not be filtered

i had reproduced on the official demo.

there its also a problem for me to hanle taxes on importing products with taxes on csv

what values i had to insert?  and what field ?

**Comment by joebordes** [comment]
sadly neither option is supported

importing taxes is easy directly in the database table:  vtiger_producttaxrel

**Comment by Netmees** [comment]
yeah... that its where i had handle it... there must be an option to filter product by taxes... so you can see if all the taxes are correct
